buildOnTravis.sh

```diff
@@ -10,6 +10,23 @@ echo "coverage   = " $2
 echo "build opts = " $3
 echo "test opts  = " $4
 
+if [ "$NATS_TEST_SERVER_VERSION" != "" ]; then
+  rel=$NATS_TEST_SERVER_VERSION
+  mkdir -p $HOME/nats-server-$rel
+  if [ "$rel" = "latest" ]; then
+    rel=$(curl -s https://api.github.com/repos/nats-io/nats-server/releases/latest | jq -r '.tag_name')
+  fi
+
+  if wget https://github.com/nats-io/nats-server/releases/download/$rel/nats-server-$rel-linux-amd64.tar.gz; then
```

Comment:
```suggestion
  if [ "$rel" != "${rel#v}" ] && wget https://github.com/nats-io/nats-server/releases/download/$rel/nats-server-$rel-linux-amd64.tar.gz; then
```
All GitHub releases seems to start with `v`.